### PR TITLE
Remove flow support

### PR DESCRIPTION
**Pull request by ranquild**
Epic: https://github.com/metabase/metabase/issues/19207

Depends on https://github.com/metabase/metabase/pull/19224 and https://github.com/metabase/metabase/pull/19236

### CODE REVIEW

File: .github/workflows/frontend.yml [hunk 1]
```diff
@@ -30,6 +30,11 @@ jobs:
       with:
         path: ~/.cache/yarn
         key: ${{ runner.os }}-yarn-${{ hashFiles('**/yarn.lock') }}
```
Comment (alxnddr) [inline .github/workflows/frontend.yml:32]: 👍 thanks for adding these

File: docs/users-guide/06-sharing-answers.md [hunk 2]
```diff
@@ -75,8 +75,8 @@ You can set up questions to run periodically and notify you if the results are i
 Next, we'll learn about how to organize our questions in [collections][collections].
 
 [alerts]: 15-alerts.md
-[archiving-items]: collections.md#archiving-items
```
Comment (alxnddr) [inline docs/users-guide/06-sharing-answers.md:78]: These markdown changes look unrelated

**Review by alxnddr — APPROVED** [review]
Lgtm! Please revert markdown changes before merging

File: docs/users-guide/06-sharing-answers.md (hunk 2 above)
Comment (ranquild) [reply docs/users-guide/06-sharing-answers.md]: Fixed
